RandLAPACK/misc/rl_linops.hh

```diff
@@ -15,6 +16,130 @@
 
 namespace RandLAPACK::linops {
 
+// Abstract linear operator concept
+template<typename LinOp, typename T = LinOp::scalar_t>
+concept LinearOperator = requires(LinOp A) {
+    { A.n_rows }  -> std::same_as<const int64_t&>;
+    { A.n_cols }  -> std::same_as<const int64_t&>;
+} && requires(LinOp A, Layout layout, Op trans_A, Op trans_B, int64_t m, int64_t n, int64_t k, T alpha, int64_t lda, T* const B, int64_t ldb, T beta, T* C, int64_t ldc) {
+    // A Matmul-like function that updates C := alpha A*B + beta C, where
+    // B and C have n columns and are stored in layout order with strides (ldb, ldc).
+    { A(layout, trans_A, trans_B, m, n, k, alpha, lda, B, ldb, beta, C, ldc) } -> std::same_as<void>;
```

Comment:
This signature shouldn't accept ``lda``; ``lda`` only has meaning for dense matrices.

Reply:
Fixed.